### PR TITLE
Improve loading time of Profile page (#1394)

### PR DESCRIPTION
**Pull request by marcinbodnar**

### CODE REVIEW

**Comment by Patrick1904** [comment]
@marcinbodnar can we please load the 'in staking pools' balances after the initial account balance has loaded vs loading them when the user expands the accordion? That way the initial balance still loads instantly and the user doesn't have to wait for staking pool balances if they expand 'in staking pools'

**Comment by vgrichina** [comment]
Somewhat related https://github.com/near/near-api-js/pull/492 – we can potentially cut some requests at near-api-js level I think.

**Comment by marcinbodnar** [comment]
> @marcinbodnar can we please load the 'in staking pools' balances after the initial account balance has loaded vs loading them when the user expands the accordion? That way the initial balance still loads instantly and the user doesn't have to wait for staking pool balances if they expand 'in staking pools'

@Patrick1904 done, please test, it wasn't so trivial as I thought

File: src/components/common/Accordion.js [hunk 1]
```diff
@@ -29,6 +29,8 @@ const Accordion = ({ className, trigger, children }) => {
     }, [open])
 
     const handleClick = () => {
+        action && action()
```
Comment (Patrick1904) [inline src/components/common/Accordion.js:32]: @marcinbodnar can we revert the changes to `Accordion`?

Comment (marcinbodnar) [reply src/components/common/Accordion.js:32]: @Patrick1904 I left it on purpose, it might be handy in the future to handle action by this component

**Review by Patrick1904 — APPROVED** [review]
LGTM
